projects/dendrites/permutedMNIST/run.py

```diff
@@ -33,29 +33,43 @@
 
 from experiments import CONFIGS
 from nupic.research.frameworks.vernon.parser_utils import DEFAULT_PARSERS, process_args
-from nupic.research.frameworks.vernon.run_with_raytune import run
+from nupic.research.frameworks.vernon.run import run as run
+from nupic.research.frameworks.vernon.run_with_raytune import run as run_with_ray_tune
 
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(
         parents=DEFAULT_PARSERS,
     )
-    parser.add_argument("-e", "--experiment", dest="name", default="default_base",
-                        help="Experiment to run", choices=CONFIGS.keys())
+
+    # TODO
```

Comment:
Maybe get rid of this TODO?

Reply:
Yes, that to-do was implemented. Good catch.